apps/console/src/components/pages/protected/tasks/create-task/dialog/delete-task-dialog.tsx

```diff
@@ -36,7 +36,12 @@ const DeleteTaskDialog: React.FC<{ taskName: string }> = ({ taskName }) => {
       <Button icon={<Trash2 />} iconPosition="left" variant="outline" onClick={() => setIsOpen(true)}>
         Delete
       </Button>
-      <ConfirmationDialog open={isOpen} onOpenChange={setIsOpen} onConfirm={handleDelete} description="This action cannot be undone, this will permanently remove the task from the organization." />
+      <ConfirmationDialog
+        open={isOpen}
+        onOpenChange={setIsOpen}
+        onConfirm={handleDelete}
+        description={`This action cannot be undone, this will permanently remove the task ${taskName} from the organization.`}
```

Comment:
descriptions are generic everywhere, we are not passing names or id-s anywhere else in the app
```suggestion
        description="This action cannot be undone, this will permanently remove the task from the organization."
```